### PR TITLE
created awesome books in es6

### PR DESCRIPTION
**Pull request by Becky449**
##Created modules which have three files for the different classes
##Use luxon.js for the es6 and time updates
##aligned all the files in relation to the instruction given i.e index.html, index.css, index.js

### CODE REVIEW

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @Becky449, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Try-Again

> ## (Highlights) Good Points: 💪 
> * Good commit messages. :heavy_check_mark:
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:

## Required Changes ♻️
- [ ] I suggest you kindly edit your [README](https://github.com/microverseinc/readme-template) file in your repo, using the Microverse [template](https://github.com/microverseinc/readme-template) as this will give users detailed information about your project. 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Nothing to mention.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1 +1,63 @@
...
+[Live Demo Link](https://becky449.github.io/Modules/)
+
+
+
+### Prerequisites
+
+- node.js
+- git
+
+### Install
```
Comment (FrederickMih) [inline README.md:29]: - [ ] To make your README.md file more descriptive, Kindly provide detailed instructions on how to set up your app in my local environment.

File: README.md [hunk 2]
```diff
@@ -1 +1,63 @@
...
+
+- GitHub: [@github](https://github.com/becky449)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
```
Comment (FrederickMih) [inline README.md:56]: - [ ] I think there's a header after this one called **Acknowledgments** that you are currently missing in your README file. Please include it to render your README more descritpive.

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi @Becky449 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.




## Highlights

- PR has a good title :+1: 
- No Linter error :+1: 



 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**